backend/trojan_dining/send_text_alert.py

```diff
@@ -2,7 +2,59 @@
 import os
 from twilio.rest import Client
 
-def send_text_alert(item_name, dining_hall, phone_number):
+def send_text_alert(association_map, phone_number):
+
+    def construct_text_notification():
+
+        dining_map = {"USC Village Dining Hall": "Village", "Parkside Restaurant & Grill": "Parkside", "Everybody's Kitchen": "EVK"}
```

Comment:
Do we need to define `construct_text_notification` inside of `send_text_alert`? I was thinking it would just be another function in the file